### ISSUE TITLE
Upgrade android dev tools: android studio 4.0.1, android.gradle plugin-4.0.1, gradle-6.5.1, sdk build tool 29.0.2

### ISSUE THREAD

**Issue by aismann** [c4games/engine-x#148]
See title

**Comment by aismann** [comment]
**For the documentation on Windows OS/Android Studio deployment:**
Set an environment variable ANDROID_NDK_HOME=<path of android-ndk>
=> Android build is running out of the box (on my developer environment)

BUT: I have to remove this  .so from src/man/jniLibs/   <<<< should be removed via build/not part of the 3d party stuff

**Comment by aismann** [comment]
Hint for Android build out of the box:
Remove *.so from src/man/jniLibs/ before starting the build.

**Comment by halx99** [comment]
* Can easy to auto package .so to apk
